docs/prow/migration-guide.md

```diff
@@ -67,39 +70,38 @@ presubmits:
+| **path_alias**              | The non-standard Go path under which you want to clone the test repository.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| **extra_ref**               | Additional repositories to clone in addition to the main repository.                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| **spec.containers.image**   | A Docker image. The `test-infra` repository provides Go and Node.js images. For more details, go to `prow/images`. To build a Docker image in your job, you require the privileged mode.                                                                                                                                                                                                                                                                                              |
+| **spec.containers.command** | Buildpacks have the `build.sh` script that allows you to easily trigger the build for your component. The `build.sh` script populates some environment variables, such as **DOCKER_TAG**. It also initializes Docker and executes one of the `Makefile` targets, depending on the value of the **BUILD_TYPE** variable. This environment variable can be injected by one of the build's Presets, which are **preset-build-pr**, **preset-build-release**, or **preset-build-master**. |
+| **spec.containers.args**    | The `build.sh` script requires path to the directory with the location of the `Makefile` for your component.                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Something is wrong with this sentence, looks like @kazydek suggestion is not applied.